main.py

```diff
@@ -33,7 +33,7 @@
 
 # iterate through cartesian product of hyperparameters and run experiment
 hparam_sets = list(ph.gen_hparam_cartesian_product())
-for i in range(ph['curr_run_number'], len(hparam_sets)):
+for i in range(ph['run_number'], len(hparam_sets)):
```

Comment:
If you're going to change curr_run_number to run_number you have to make sure you've changed it everywhere its called in the repo

Reply:
curr_run_number was throwing an error

Reply:
supervised_hp.py had the wrong config parameter I believe; should be curr_run_number not run_number?